src/tests/ftest/rebuild/no_cap.yaml

```diff
@@ -12,8 +12,7 @@ server_config:
   name: daos_server
 # reduce cart timeout to make IV update return timeout
 # quickly to ease test (DAOS-9626)
-  env_vars:
-    - CRT_TIMEOUT=10
+  crt_timeout: 10
```

Comment:
Does no_cap.py need to be modified to pick up this value?

Reply:
No, see this example, crt_timeout is applied correctly in the daos server yml file:
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8337/1/artifact/Functional%20Hardware%20Medium/rebuild/no_cap.py/1/daos_configs/daos_server.wolf-122.yml/*view*/

Reply:
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8337/4/artifact/Functional%20Hardware%20Medium/rebuild/no_cap.py/9/daos_logs/daos.wolf-52.log shows that the client CRT_TIMEOUT is indeed set to 10 s.

Reply:
Client CRT_TIMEOUT seems picked from server side config, apricot/apricot/test.py:            crt_timeout=self.server_managers[index].get_config_value("crt_timeout"))